openpype/settings/entities/schemas/projects_schema/schema_project_max.json

```diff
@@ -65,6 +65,104 @@
                 }
             ]
         },
+        {
+            "type": "dict",
+            "collapsible": true,
+            "key": "PreviewAnimation",
```

Comment:
I believe if you make this key the exact class name `CreateReview` you don't need the `apply_settings` implementation since it then defaults to applying these from settings. At least, that's what I'm seeing in many other plug-ins being done. Having a matching key name does make it clearer that these are linked?

Thoughts?

@iLLiCiTiT or are we trying to move away from that implicit inheriting of settings?

Reply:
At this moment it does not apply automatically for create plugins, we'll probably add it, because devs implemented it anyways.

Neverthelss I think it should be named `CreateReview`.

Reply:
Already Renamed!